docs/Tutorials/BeginnersTutorial.md

```diff
@@ -0,0 +1,547 @@
+or just a terminal). First we will need to run `cmake` to set up everything
+because we use CMake as our build system.
+
+### With a Terminal {#with_terminal}
+
+Inside the SpECTRE repository, make a directory called `build`. This is where
+all the compiled objects will go. `cd` into the `build` directory. To configure
+SpECTRE in Release mode (which is recommended for running simulations), run the
+following command
+
+
+```
+cmake -D CMAKE_BUILD_TYPE=Release \
+      -D CHARM_ROOT=/work/charm_7_0_0/multicore-linux-x86_64-clang \
+      -D CMAKE_CXX_COMPILER=/usr/bin/clang++-10 \
+      -D CMAKE_C_COMPILER=/usr/bin/clang-10 ../
+```
+
+[Charm++](http://charmplusplus.org/) is the library we use for task-based
+parallelism. The `../` is the path to the root of the SpECTRE directory
+(relative or absolute, it doesn't matter). This should only take a few seconds
+to run.
+
+Now that everything is configured, you can build the
+`ExportTimeDependentCoordinates3D` executable like so
+
+```
+make ExportTimeDependentCoordinates3D -jN
+```
+
+Here, `N` is the number of cores you'd like to use to build. It's entirely up to
+you on how many you'd like to use. Since you are most likely running this on a
+laptop, `-j4` should be sufficient.
+
+### With VSCode
+
+If you are using VScode, open the
+[command palette]
+(https://code.visualstudio.com/docs/getstarted/userinterface#command-palette)
+and run the following commands:
+
+1. `CMake: Select a Kit` - Select the `Default` kit
+2. `CMake: Select Variant` - Select the `Release` variant to make the code run
+   faster.
+3. `CMake: Configure` - This will create a build directory called
+   `build-Default-Release` and run an appropriate `cmake` command for you
+   automatically.
+4. `CMake: Build Target` - Select the `ExportTimeDependentCoordinates3D`
+   executable.
+
+This will most likely use all the cores on your laptop to build the executable.
+If configuring and building with VSCode doesn't work, just open a terminal and
+do it manually as in the \ref with_terminal section.
+
+Once the executable is built, you're ready to run it!
+
+## Running ExportTImeDependentCoordinates3D
+
+To run an executable that you built inside the container, you must remain inside
+the container. Make a directory `runs` inside your build directory where you
+will run everything. Inside `runs`, sym-link the executable you just build into
+this directory like so
+
+```
+ln -s ../bin/ExportTimeDependentCoordinates3D .
+```
+
+The reason we have you sym-link is just so that you don't have to copy over the
+executable every time you rebuild.
+
+Next, copy over the input file
+`tests/InputFiles/ExportCoordinates/InputTimeDependent3D.yaml` into your `runs`
+directory. We will be visualizing the coordinates of a binary black hole domain.
+To run the executable, do
+
+```
+./ExportTimeDependentCoordinates3D --input-file InputTimeDependent3D.yaml
+```
+
+This will run it on one core. If you'd like to use more cores, add the `++ppn N`
```

Comment:
For non-SMP mode I don't think it matters. I chose `++ppn N` because we use that in most of our submit scripts so if people look at those they won't see an unfamiliar option.